bedrock-devnet/devnet/__init__.py

```diff
@@ -300,6 +360,83 @@ def devnet_deploy(paths):
         log.info('Bringing up `da-server`, `sentinel`.') # TODO(10141): We don't have public sentinel images yet
         run_command(['docker', 'compose', 'up', '-d', 'da-server'], cwd=paths.ops_bedrock_dir, env=docker_env)
 
+
+def deploy_l2_interop(paths):
+    generate_l2_genesis(INTEROP_CHAIN_ID,
+                        paths.interop_genesis_l2_path, paths.interop_l1_deployments_path,
+                        paths.interop_devnet_config_path, paths.interop_addresses_json_path,
+                        paths.interop_rollup_config_path, paths, interop=True)
+    rollup_config = read_json(paths.interop_rollup_config_path)
+    addresses = read_json(paths.interop_addresses_json_path)
+
+    # Start the L2.
+    log.info('Bringing up L2.')
+    run_command(['docker', 'compose', 'up', '-d', 'l2-interop'], cwd=paths.ops_bedrock_dir, env={
+        'PWD': paths.ops_bedrock_dir
+    })
+
+    # Wait for the L2 to be available.
+    wait_up(10545)
+    wait_for_rpc_server('127.0.0.1:10545')
+
+    # Print out the addresses being used for easier debugging.
+    l2_output_oracle = addresses['L2OutputOracleProxy']
+    dispute_game_factory = addresses['DisputeGameFactoryProxy']
+    batch_inbox_address = rollup_config['batch_inbox_address']
+    log.info(f'Using L2OutputOracle {l2_output_oracle}')
+    log.info(f'Using DisputeGameFactory {dispute_game_factory}')
+    log.info(f'Using batch inbox {batch_inbox_address}')
+
+    # Set up the base docker environment.
+    docker_env = {
+        'PWD': paths.ops_bedrock_dir,
+    }
+
+    # Selectively set the L2OO_ADDRESS or DGF_ADDRESS if using L2OO.
+    # Must be done selectively because op-proposer throws if both are set.
+    if DEVNET_L2OO:
```

Comment:
We could simplify here and force fault proofs and no plasma

Reply:
done